src/partials/_contacts.html

```diff
@@ -1,28 +1,31 @@
 <section class="contacts">
   <div class="contacts__background">
-    <h2 class="contacts__section-title">Contacts</h2>
+    <p class="contacts__sectiontitle">Contacts</p>
     <div class="contacts_flex">
       <!-- karta 1 -->
-      <ul class="contacts__cards">
+      <ul class="contacts__card">
```

Comment:
tu chodziło mi bardziej o to żeby specjalna lista z klasą np contacts__card-list oplatała wszystkie 3 karty, i każda kolejna karta powinna być jej elementem, a w środku każdej karty powinna znajdować nie kolejna lista z elementami

Reply:
"nie kolejna lista z elementami" nie wiem jak to rozumieć, reszta zrozumiała. Czyli lista kart a w środku nie listy tylko luźno elementy ?